mainwindow.cpp

```diff
@@ -94,6 +99,14 @@ namespace dealii
 
 
 
+    void MainWindow::set_to_default()
+    {
+      QTreeWidgetItem * current_item = tree_widget->currentItem();
+      current_item->setText(1,current_item->text(2));
```

Comment:
Yes, I chose the place to ask my question poorly -- but there are all of the other places where you/the previous author create objects with `new`.

No need for an issue. I was just curious. If we create a memory leak, then that's aesthetically not pleasing, but I'm not overly concerned about it in the current program.